hotline/species/indexes.py

```diff
@@ -0,0 +1,19 @@
+from elasticmodels import Index, StringField
+
+from .models import Species
+
+
+class SpeciesIndex(Index):
+
+    severity = StringField(attr="get_severity")
```

Comment:
You could do `StringField(attr="severity.name")` and you wouldn't have to write the "get_severity" method. This could probably be clearer in the docs: https://github.com/PSU-OIT-ARC/elasticmodels#field-classes